Cargo.toml

```diff
@@ -11,6 +11,7 @@ exclude = [".github/workflows/dusk-ci.yml", ".gitignore"]
 [dependencies]
 rand_core = { version = "0.6", default-features = false }
 dusk-bytes = "0.1"
+dusk-plonk = { version = "0.19", default-features = false, optional = true }
```

Comment:
If dusk-plonk is needed only in tests, should this dependency be included only as dev-dependency? I think it should but pls. correct me if there is a reason not to.

Reply:
plonk is not only used in tests, is used in the gadget module to encrypt in ZK